resource_catalogue_fastapi/utils.py

```diff
@@ -212,9 +216,14 @@ def execute_order_workflow(
             "workspace": user_workspace,
             "stac_key": stac_key,
             "workspace_bucket": workspace_bucket,
+            "workspaces_domain": workspaces_domain,
+            "env": "prod",
         }
     }
 
+    logger.info(f"Sending request to {url} with payload: {payload}")
+    logger.info(f"Headers: {headers}")
```

Comment:
Debug logging to remove

Reply:
Is logging bad?

Reply:
I'll keep the first line, but the headers will contain a user token and should not be logged long-term